common/shared_config.py

```diff
@@ -39,6 +39,8 @@
 serper_api_key = ''
 random_seed = 1
 model_options = {
+    'gpt_4o_mini': 'OPENAI:gpt-4o-mini',
```

Comment:
same as above, point to `'OPENAI:gpt-4o-mini-2024-07-18'`

Reply:
pinned version here as well